### PR TITLE
Adding requirements

### PR DESCRIPTION
**Pull request by mexanick**
## Description

Provoding requirements for pip and rpm packaging
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by bdorney — DISMISSED** [review]

**Comment by jsturdy** [comment]
> Out of date with base branch

Not necessarily a problem, provided the `merge` button still works, it will do the required actions to `rebase` or `merge` which is the point of using `rebase`ing in the workflow

File: python/reg_interface/pkg/setup.cfg [hunk 1]
```diff
@@ -50,4 +50,6 @@ exclude=build,lib,.tox,third,*.egg,docs,packages,.eggs
 
 [bdist_rpm]
 requires = python >= 2.6
+    reg_utils-rwreg
```
Comment (jsturdy) [inline python/reg_interface/pkg/setup.cfg]: is this something you're changing with the update to the `specTemplate`?

Comment (mexanick) [reply python/reg_interface/pkg/setup.cfg]: fixed

**Comment by mexanick** [comment]
merge conflicts resolved

**Review by jsturdy — APPROVED** [review]

**Pull request merged**